### PR TITLE
Feat/landsat api logic

### PR DESCRIPTION
**Pull request by wikipop**
resolves (#16)

### CODE REVIEW

**Comment by wikipop** [comment]
ik ik this gonna be squash

File: pyproject.toml [hunk 1]
```diff
@@ -18,6 +18,8 @@ structlog = "^24.4.0"
 sqlalchemy = "^2.0.35"
 psycopg2 = "^2.9.9"
 pydantic-settings = "^2.5.2"
+pytest = "^8.3.3"
```
Comment (Norbiros) [inline pyproject.toml:21]: Move pytest to dev dependencies

File: package.json [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
     "format": "pnpm run /:format$/",
-    "test": "echo silly tests",
+    "test": "pnpm run /:test$/",
     "postinstall": "poetry install && poetry run openapi",
     "prepare": "husky"
   },
   "devDependencies": {
     "husky": "^9.1.6"
-  },
-  "packageManager": "pnpm@9"
```
Comment (Norbiros) [inline package.json]: Why you removed that?

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://user:password@localhost:5432/landsat_web_tracker_db
+NUXT_OPEN_FETCH_PETS_BASE_URL=http://localhost:8000
```
Comment (Norbiros) [inline .env.example:2]:
```suggestion
NUXT_OPEN_FETCH_API_BASE_URL=http://localhost:8000
```

File: package.json (hunk 2 above)
Comment (wikipop) [reply package.json]: silly

**Pull request closed without merging**